### PR TITLE
Clean up images - rename (issue 1261)

### PR DESCRIPTION
**Pull request by Lennonka**
Issue #1261 

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3

### CODE REVIEW

File: guides/common/modules/proc_implementing-pxe-less-discovery.adoc [hunk 1]
```diff
@@ -21,15 +21,15 @@ The console might freeze during the process.
 On some hardware, you might experience graphical hardware problems.
 
 ifdef::satellite[]
-image::common/PXEless-mode.png[]
+image::common/pxeless-mode.png[]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_implementing-pxe-less-discovery.adoc:24]:
```suggestion
image::common/pxeless-mode-satellite.png[]
```

Please also rename the file itself.

**Review by maximiliankolb — COMMENTED** [review]
I would like to not loose the `.drawio` files. Maybe we can instead move them to another directly that will not be part of built docs?

**Comment by maximiliankolb** [comment]
Comparing the built docs I was expecting a smaller archive:

Screenshot 2022-09-27 at 11 49 04
Screenshot 2022-09-27 at 11 48 57

File: guides/common/modules/proc_implementing-pxe-less-discovery.adoc (hunk 1 above)
Comment (Lennonka) [reply guides/common/modules/proc_implementing-pxe-less-discovery.adoc]: Done, thanks for the catch!

**Comment by maximiliankolb** [comment]
Me too. I am hesitant to merging this (with good intention) when the size almost doubles. Maybe we need to introduce a way to split images by build target?

I think `guides/image-sources` is a good place.

**Comment by ekohl** [comment]
This is precisely what I mentioned in https://github.com/theforeman/foreman-documentation/issues/1261#issuecomment-1253798839. Because the images now get places in every resulting guide (regardless if they're used or not) the size is much larger than if they only reside in the guide that actually uses them.

I've been wondering if we should bite the bullet and address https://github.com/theforeman/foreman-documentation/issues/873. Hopefully Antora is smarter about image placement, though I don't have any idea if this is really true.

**Comment by maximiliankolb** [comment]
I've just checked on our Antora instance: Images that are included multiple times are only on there once which is nice.

File: guides/image-sources/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Image sources
+
+This folder contains editable sources of images.
+
+Draw.io files belong here.
+
+Exported images in PNG and SVG formats go to `common/images`.
```
Comment (maximiliankolb) [inline guides/image-sources/README.md:7]:
```suggestion
Exported images in PNG and SVG formats go to `guides/common/images`.
```

**Comment by Lennonka** [comment]
I've modified it to only rename the image files, but keep their location.

**Review by maximiliankolb — APPROVED** [review]
Thanks, LGTM!

I think it'd be helpful if you'd add a sentence or two on **where** and **how** to place images in general to our conventions/README/CONTRIBUTING (whatever you see fit best). Feel free to do this is another small follow-up PR.

**Comment by Lennonka** [comment]
Cherry-picked:
- 0216b5b6f..c633f8129  3.4 -> 3.4
- 61ace1c74..6ad4b29a8  3.3 -> 3.3
- 3b389cf10..90406e5b2  3.2 -> 3.2
- c5be41cd9..9912194dd  3.1 -> 3.1

**Pull request merged**